project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.html

```diff
@@ -400,336 +353,200 @@
+          </div>
+          <div class="flex">
+            <mat-icon class="ws-mat-default-text cursor-pointer" (click)="toggleCreds()">{{showCreds ? 'keyboard_arrow_up'
+              : 'keyboard_arrow_down'}}</mat-icon>
           </div>
         </div>
-        <mat-card-subtitle role="link" tabindex="0" i18n-aria-label>
-          {{ portalProfile?.employmentDetails?.departmentName }}
-        </mat-card-subtitle>
-
-        <mat-card-subtitle role="link" tabindex="0" i18n-aria-label
-          *ngIf="portalProfile?.additionalProperties?.externalSystem === 'DoPT eHRMS'">
-          eHRMS Id - {{ portalProfile?.additionalProperties?.externalSystemId }}
-        </mat-card-subtitle>
-        <ng-container>
-          <div class="flex flex-1 custom margin-0 margin-top-m bottom-fix mob" role="link" tabindex="0">
-            <div class="flex flex-1">
-              <span class="flex margin-right-l mobile" *ngIf="isCurrentUser">
-                <span class="ws-mat-default-text">
-                  <img class="karmapoints-icon" alt="karma points" src="/assets/icons/karma-point-logo.jpg">
-                </span>
-                <span class="ws-mat-default-text">
-                  {{ countdata }}</span>
-                <span class="pad-4"> {{ 'profileView.karmaPoints' | translate }}</span>
-              </span>
-              <span class="flex margin-right-l mobile">
-                <span class="ws-mat-default-text">{{
-                  connectionRequests?.length
-                  }}</span>
-                <span class="pad-4">{{ 'profileView.connections' | translate }}</span>
-              </span>
-              <span class="flex margin-right-l mobile">
-                <span class="ws-mat-default-text">{{
-                  Math.abs(discussProfileData?.topiccount || 0)
-                  }}</span>
-                <span class="pad-4">{{ 'profileView.discussions' | translate }}</span>
-              </span>
+        <div class="my-credentials p-3" *ngIf="showCreds">
+          <div class="flex flex-middle" style="padding: 5px 0px;">
+            <span class="pr-5 font-semibold">{{ 'profileInfo.userName' | translate }}</span>
+            <div class="flex flex-middle copy-items">
+              <span class="pr-2 custom-width">{{ assessmentsData?.userName }}</span>
+              <span class=""><mat-icon class="cursor-pointer copy-icon"
+                  (click)="copyToClipboard('Venkat')">content_copy</mat-icon></span>
+            </div>
+          </div>
+          <div class="flex flex-middle" style="padding: 5px 0px;">
+            <span class="pr-6 font-semibold">{{ 'profileInfo.password' | translate }}</span>
+            <div class="flex flex-middle copy-items">
+              <span class="pr-2  custom-width">{{ assessmentsData?.password }}</span>
+              <span class=""><mat-icon class="cursor-pointer copy-icon"
+                  (click)="copyToClipboard('Test@1234')">content_copy</mat-icon></span>
```

Comment:
Is it will be like this only?

Reply:
Yes! Checked with @venkykandagaddala  he mentioned to keep code as it is